data/mods/gen4/abilities.js

```diff
@@ -402,7 +402,11 @@ let BattleAbilities = {
 		inherit: true,
 		shortDesc: "This Pokemon is only damaged by Fire Fang, supereffective moves, indirect damage.",
 		onTryHit(target, source, move) {
-			if (target === source || move.category === 'Status' || move.type === '???' || move.id === 'struggle' || move.id === 'firefang') return;
+			if (move.id === 'firefang') {
+				this.hint('In Gen 4, Fire Fang is always able to hit through Wonder Guard.');
```

Comment:
```suggestion
				this.hint("In Gen 4, Fire Fang is always able to hit through Wonder Guard.");
```

In case it was unclear, PS's standard is to use `'` for IDs, `` ` `` for code and strings that need to be interpolated, and `"` for all other strings.

And yes, this standard is violated all over the place, but I do try to police it when I remember.

Reply:
https://github.com/Zarel/Pokemon-Showdown/blob/master/CONTRIBUTING.md may be useful to you, if you haven't read that yet.

Reply:
I mostly just tried to follow what the code around it was doing.